plugins/modules/pnp_intent.py

```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
             self.module.fail_json(msg="Site not found", response=[])
 
         if response:
-            self.log(str(response))
+            self.log("Site details of {0} received are {1}".format(self.want.get("site_name"), str(response)), "DEBUG")
             site = response.get("response")
             site_additional_info = site[0].get("additionalInfo")
             for item in site_additional_info:
                 if item["nameSpace"] == "Location":
                     site_type = item.get("attributes").get("type")
+                    self.log("Site Type is {0}".format(site_type), "INFO")
```

Comment:
self.log("Site type for site name '{1}' : {0}".format(site_type, site_name), "INFO")

Reply:
Resolved